src/client/java/daos-java/find_build_paths.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+daos_install_path=$1
+var=$2
+build_vars=../../../../.build_vars.sh
+if [ ! -f "${build_vars}" ]
+then
+  if [ "$var" = "SL_PREFIX" ]
+  then
+    echo "${daos_install_path}"
+  else # SL_PROTOBUFC_PREFIX, default to dev
```

Comment:
should probably default to release per SDL requirements.

Reply:
Should this be fixed prior to landing, given that it's an SDL requirement?

Reply:
@jolivier23 @brianjmurrell I changed the default to release.